docs/contributing.md

```diff
@@ -71,6 +144,17 @@ tested for compliance with the coding style.
 To speed up a review from the project's maintainers, please make sure that
 the CI checks are passing for your PR.
 
+## Complex contributions
+
+If you have a complex contribution in mind (meaning changes in the architecture
+or a lot of LOC changed), it is advisable to first file a GitHub issue and
+discuss the implementation with the project's maintainers via [#headlamp](https://kubernetes.slack.com/messages/headlamp) Slack channel.
+
+## Translations
+
+If you want to contribute to the internationalization of Headlamp, please refer to the
+dedicated [i18n docs](./development/).
```

Comment:
Missed the i18n part I think: `./development/i18n/`

Reply:
done!